storm-server/src/main/java/org/apache/storm/daemon/nimbus/Nimbus.java

```diff
@@ -2363,7 +2363,7 @@ private CommonTopoInfo getCommonTopoInfo(String topoId, String operation) throws
             ret.launchTimeSecs = 0;
         }
         ret.assignment = state.assignmentInfo(topoId, null);
-        ret.beats = Utils.OR(heartbeatsCache.get().get(topoId), Collections.emptyMap());
+        ret.beats = Utils.<Map<List<Integer>, Map<String, Object>>>OR(heartbeatsCache.get().get(topoId), Collections.emptyMap());
```

Comment:
I am getting the below error when I do a `mvn clean install` in the storm-server module without the type hints. Will check again.

```
[ERROR] COMPILATION ERROR :
[INFO] -------------------------------------------------------------
[ERROR] /Users/aiyer/github/storm/storm-server/src/main/java/org/apache/storm/daemon/nimbus/Nimbus.java:[2366,29] incompatible types: inferred type does not conform to upper bound(s)
    inferred: java.util.Map<? extends java.lang.Object,? extends java.lang.Object>
    upper bound(s): java.util.Map<java.util.List<java.lang.Integer>,java.util.Map<java.lang.String,java.lang.Object>>,java.lang.Object

$ java -version
java version "1.8.0_51"
Java(TM) SE Runtime Environment (build 1.8.0_51-b16)
Java HotSpot(TM) 64-Bit Server VM (build 25.51-b03, mixed mode)
```